File: app/src/androidTest/java/com/neocaptainnemo/atms/mocks/MockServiceModule.java
package com.neocaptainnemo.atms.mocks;

import com.neocaptainnemo.atms.service.Atms;

import javax.inject.Singleton;

import dagger.Module;
import dagger.Provides;

import static org.mockito.Mockito.mock;

@Module
class MockServiceModule {

    @Provides
    @Singleton
    Atms providesAtms() {

        return mock(Atms.class);
    }

}

File: app/src/androidTest/java/com/neocaptainnemo/atms/mocks/MockServiceModule.kt
package com.neocaptainnemo.atms.mocks

import com.neocaptainnemo.atms.service.Atms
import dagger.Module
import dagger.Provides
import org.mockito.Mockito.mock
import javax.inject.Singleton

@Module
class MockServiceModule {

    @Provides
    @Singleton
    fun providesAtms(): Atms = mock(Atms::class.java)
}

